packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,257 @@
+		} );
+	} else {
+		if ( pathsToCommit.length ) {
+			console.log( '\n📍 ' + chalk.blue( 'Committing the changes...\n' ) );
+
+			for ( const path of pathsToCommit ) {
+				const execOptions = {
+					stdio: 'inherit',
+					cwd: path
+				};
+
+				console.log( `${ chalk.green( '+' ) } ${ path }` );
+
+				execSync( `git add ${ path }`, execOptions );
+				execSync( 'git commit -m "Internal: Updated all CKEditor 5 dependencies ' +
+				'in `packages/*` to the latest version. [skip ci]"', execOptions );
+			}
+
+			console.log( '\n📍 ' + chalk.green( `Successfully committed ${ totalResult.toCommit } files!\n` ) );
+		}
+
+		if ( totalResult.updated ) {
+			console.log( '\n📍 ' + chalk.green( `Updated total of ${ totalResult.updated } files!\n` ) );
+		} else {
+			console.log( '\n📍 ' + chalk.green( 'No files needed an update.\n' ) );
+		}
+	}
+};
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in the specified directory to the latest version.
+ * Latest version is taken from the `version` property from the root of the `package.json` file, since that value
+ * should already have been bumped at this point of release process.
+ *
+ * @param {String} pathToUpdate Directory containing files to update.
+ * @param {Boolean} dryRun If set to true, diff of changes that would be made is calculated, and included in the returned object. Without
+ * this flag, file is updated normally.
+ * @returns {UpdateResult}
+ */
+function updateDirectory( pathToUpdate, dryRun ) {
+	const globPattern = pathToUpdate + '/*/package.json';
+	const packageJsonArray = glob.sync( globPattern );
+
+	if ( !packageJsonArray.length ) {
+		return { found: 0, updated: 0, differences: [] };
+	}
+
+	let updatedFiles = 0;
+	const differences = [];
+
+	for ( const file of packageJsonArray ) {
+		const currentFileData = fs.readFileSync( file, 'utf-8' );
+		const parsedData = JSON.parse( currentFileData );
+
+		updateObjectProperty( parsedData, 'dependencies' );
+		updateObjectProperty( parsedData, 'devDependencies' );
+
+		const newFileData = JSON.stringify( parsedData, null, 2 ) + '\n';
+
+		if ( currentFileData !== newFileData ) {
+			updatedFiles++;
+
+			if ( dryRun ) {
+				differences.push( {
+					file,
+					content: diff( currentFileData, newFileData, { newlineIsToken: true } )
+				} );
+			} else {
+				fs.writeFileSync( file, newFileData, 'utf-8' );
+			}
+		}
+	}
+
+	return { found: packageJsonArray.length, updated: updatedFiles, differences };
+}
+
+/**
+ * Updates the CKEditor 5 dependencies, except the `*-dev` and `*-inspector` ones, in the provided `package.json` file.
+ *
+ * @param {Object} parsedPkgJson Object to update.
+ * @param {String} propertyName Name of the property to update.
+ */
+function updateObjectProperty( parsedPkgJson, propertyName ) {
+	const version = parsedPkgJson.version;
+	const regex = /^@ckeditor\/ckeditor5-([a-z]+)|^ckeditor5$/;
+	const exceptions = [
+		'dev',
+		'inspector',
+		'react',
+		'vue',
+		'angular'
+	];
```

Comment:
For me, it would be fine to just add the exceptions to the regexp:

```js
const regex = /^@ckeditor\/ckeditor5-(?!dev|inspector|react|vue|angular)|^ckeditor5$/;
```

Decide for yourself which is more readable.

Reply:
IMO regexes are hard to read by their nature, and thus array with an explicit name is way better. Also, if for any reason we need to add more in the future, array is more expandable. Let's keep it the way it is.

Reply:
I'd go with supporting strings and regexps:

```js
// Pattern that describes CKEditor 5 packages.
const CKEditor5Pattern = /^@ckeditor\/ckeditor5-(.*)|^ckeditor5$/; // [a-z]+ does not catch `ckeditor5-word-count`.

// Names of patterns to ignore.
const patternsToSkip = [
	'@ckeditor/ckeditor5-inspector',
	/@ckeditor/ckeditor5-dev-.*/, // All dev utils. Including the private too.
	'@ckeditor/jsdoc-plugins',
	/* ... */
];
```

Reply:
I believe that exception for `'@ckeditor/jsdoc-plugins',` is not needed, as `/^@ckeditor\/ckeditor5-(.*)|^ckeditor5$/` will not catch it anyway